src/ResourceManager/KeyVault/Commands.KeyVault/Commands.KeyVault.csproj

```diff
@@ -49,7 +51,37 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="Commands\AddAzureKeyVaultCertificate.cs" />
+    <Compile Include="Commands\SetAzureKeyVaultCertificateAttribute.cs" />
+    <Compile Include="Commands\StopAzureKeyVaultCertificateOperation.cs" />
```

Comment:
Stop is a paired verb with start: https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx
Is it ok from cmdline design perspective that there is not 'start' but there is a 'stop' command? How does the polling happen? In general what is the pattern used for long running operations in Azure PowerShell?